pom.xml

```diff
@@ -0,0 +1,666 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>4.3</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.github.ks-jenkins</groupId>
+  <artifactId>ks-jenkins</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>KubeSphere Jenkins</name>
+  <description>Jenkins formula for KubeSphere.</description>
+  <url>https://kubesphere.github.io/ks-jenkins/</url>
```

Comment:
I'm wondering if we can use `https://github.com/kubesphere/ks-jenkins` instead of the below one.

```shell
➜  ~ curl -q https://kubesphere.github.io/ks-jenkins
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
```